### PR TITLE
Optionally do not clear selection on HideIsolateEmphasizeActionHandler.processIsolateSelected

### PR DESCRIPTION
**Pull request by GintV**
## Changes

Optionally do not clear selection on HideIsolateEmphasizeActionHandler.processIsolateSelected

## Testing

N/A

### CODE REVIEW

File: ui/appui-react/src/appui-react/selection/HideIsolateEmphasizeManager.ts [hunk 1]
```diff
@@ -578,6 +580,8 @@ export class HideIsolateEmphasizeManager extends HideIsolateEmphasizeActionHandl
       HideIsolateEmphasizeActionHandler.hideIsolateEmphasizeUiSyncId
     );
 
+    if (!clearSelection) return;
```
Comment (raplemie) [inline ui/appui-react/src/appui-react/selection/HideIsolateEmphasizeManager.ts:583]: Please add a test that cover this option.

**Review by raplemie — CHANGES_REQUESTED** [review]
Please add a line in NextVersion.md identifying this new feature.

File: ui/appui-react/src/appui-react/selection/HideIsolateEmphasizeManager.ts (hunk 1 above)
Comment (GintV) [reply ui/appui-react/src/appui-react/selection/HideIsolateEmphasizeManager.ts:583]: I don't see any tests that cover any part of HideIsolateEmphasizeManager, only ui\appui-react\src\test\selection\HideIsolateEmphasizeActionHandler.test.tsx that fakes manager. Do you want to cover this param in fake manager tests? Or am I missing something?

Comment (raplemie) [reply ui/appui-react/src/appui-react/selection/HideIsolateEmphasizeManager.ts:583]: You are right, for some reason, this whole class is istanbul ignored, that shouldnt be the case but that's not on you to add these tests!

**Review by raplemie — APPROVED** [review]

File: docs/changehistory/NextVersion.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ Table of contents:
 
 ## @itwin/appui-react
 
+### Improvements
```
[the remark this answers is not in the document]
Comment (GintV) [reply docs/changehistory/NextVersion.md:19]: Done

**Pull request merged**